temporal-testing/build.gradle

```diff
@@ -0,0 +1,187 @@
+// Run 'gradle checkUpdates' to find out which dependencies have newer versions
+plugins {
+    id 'java-library'
+    id 'net.ltgt.errorprone' version '1.3.0'
+    id 'net.minecrell.licenser' version '0.4.1'
+    id 'com.palantir.git-version' version '0.12.3'
+    id 'maven-publish'
+    id 'signing'
+    id 'de.marcphilipp.nexus-publish' version '0.4.0'
+    id 'name.remal.check-updates' version '1.1.6'
+}
+
+apply plugin: 'maven-publish'
+apply plugin: 'de.marcphilipp.nexus-publish'
+apply plugin: 'java'
+
+if (hasProperty('signing.keyId')) {
+    apply plugin: 'signing'
+    signing {
+        sign configurations.archives
+    }
+}
+
+group = 'io.temporal'
+version = getVersionName()
+archivesBaseName = "temporal-testing"
+
+description = '''Temporal Workflow Java SDK testing'''
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+    withJavadocJar()
+    withSourcesJar()
+}
+
+dependencies {
+    errorproneJavac('com.google.errorprone:javac:9+181-r4173-1')
+    errorprone('com.google.errorprone:error_prone_core:2.4.0')
+
+    api project(':temporal-sdk')
+    api group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
+    api group: 'io.micrometer', name: 'micrometer-core', version: '1.6.2'
+
+    implementation group: 'com.google.guava', name: 'guava', version: '30.1-jre'
+    implementation group: 'com.cronutils', name: 'cron-utils', version: '9.1.3'
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.12.0'
+    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.12.0'
+    if (!JavaVersion.current().isJava8()) {
+        implementation 'javax.annotation:javax.annotation-api:1.3.2'
+    }
+
+    implementation group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3'
+    implementation group: 'com.googlecode.junit-toolbox', name: 'junit-toolbox', version: '2.4'
+    implementation group: 'junit', name: 'junit', version: '4.13.1'
+}
+
+configurations.all {
+    // Conflicts with logback
+    exclude group: 'org.slf4j', module: 'slf4j-simple'
```

Comment:
I don't think the testing framework needs a concrete slf4j implementation.

Reply:
Cleaned up.